certbot-nginx/certbot_nginx/_internal/configurator.py

```diff
@@ -769,9 +769,6 @@ def enhance(self, domain, enhancement, options=None):
         except (KeyError, ValueError):
             raise errors.PluginError(
                 "Unsupported enhancement: {0}".format(enhancement))
```

Comment:
Your example images look like you cut this too, but it appears to still be in the code. What's going on here?

Reply:
> It looks like you added this message? Am I missing something here? Is the question, "Can we leave this message empty, since I removed the redundant message?" If so, yes.

Sorry, I shouldn't have done two things at once, that's confusing.

I changed the status message from `f"Could not set up {enhancement} enhancement"`, which was redundant with all the  error logging, to the current message, which at least has a distinct meaning.

Further to that, my question is whether we can just not have any message at all?

> Your example images look like you cut this too, but it appears to still be in the code. What's going on here?

The reason this error is no longer being raised is because the `KeyError` was formerly being raised [here](https://github.com/certbot/certbot/blob/b150c9a2adaba72a61c44dbf9452596a5bb75492/certbot-nginx/certbot_nginx/_internal/configurator.py#L811), whereas due to [the check I added earlier in that function](https://github.com/certbot/certbot/blob/b150c9a2adaba72a61c44dbf9452596a5bb75492/certbot-nginx/certbot_nginx/_internal/configurator.py#L791-L793), it is now supplanted by a `NotSupportedError`. Thus the `except KeyError` code path is avoided.

Reply:
Ah, I see, this was meant for the `try` above, but then got implicitly reused in a weird way further down the code path. A little weird that they return different errors, but the new version is way more correct so lgtm.